cmd/image-builder/main.go

```diff
@@ -95,11 +97,14 @@ func main() {
 	}
 
 	recommendationConf := recommendations.RecommendationsClientConfig{
-		URL:          conf.RecommendURL,
-		CA:           conf.RecommendCA,
-		TokenURL:     conf.RecommendTokenURL,
-		ClientId:     conf.RecommendClientId,
-		ClientSecret: conf.RecommendSecret,
+		URL: conf.RecommendURL,
+		CA:  conf.RecommendCA,
+		Tokener: &oauth2.LazyToken{
+			Url:          conf.RecommendTokenURL,
+			ClientId:     conf.RecommendClientId,
+			ClientSecret: conf.RecommendSecret,
+			Expiration:   time.Time{},
+		},
```

Comment:
When you do not set Expiration then it is year 0000 time 0:00, therefore expired. No need to set it to current time.